src/ui/widgets/Image/image.tsx

```diff
@@ -55,10 +55,16 @@ export const ImageComponent = (
     height: imageHeight
   };
 
+  // Should we be refreshing image instead of using cache
+  let imageFileName = props.imageFile;
+  if (imageFileName.includes("#")) {
```

Comment:
It looks like we're assuming the `#` will come at the end here - in which case it may be best to make this explicit?

Reply:
The `#` will come after the .<file_extension> but might not be the last character (e.g. the filename could be image.png#1). We say that we just add `#` to the end of the filename in an OPI if we want it to be refreshed, in which case I could check the last character? Otherwise I could check that it appears after the '.' or use regex but if that is slower we have to keep in mind that this will run on every refresh?

Reply:
We have decided to keep to logic as is, I.e. does it contain a '#'. A comment has been added to the code to make it clearer why this is.